### ISSUE TITLE
[WIP] KCD 2020 Jakarta, Indonesia

### ISSUE THREAD

**Issue by irvifa** [cncf/kubernetes-community-days#239]
Please fill in information, organizers, and sponsors as you assemble them, and then make a new comment when this application is ready for review.

### Submission checklist:

*Please check each of these as you complete them:

* [x] No other Kubernetes Community Days event in the same city
* [x] Minimum of 3 organizers from 3 different organizations
* [x] Each organizer lives in or just outside of the KCD location
* [x] At least one organizer is a CNCF member employee, CNCF ambassador, or CNCF project maintainer
* [x] All organizers agree to follow the event guidelines and CNCF Code of Conduct
* [x] Minimum of 3 sponsors committed
* [x] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
* [ ] Agree to use Eventbrite for registration
* [x] Agree to use Sched for organizing the program page
* [x] Added usernames for [CNCF Slack](https://slack.cncf.io/) for all organizers so that we can add you to the #kcd-organizers channel

### Organizer 1: 
- Giri Kuncoro, Gojek, CNCF Ambassador/Software Engineer, girikuncoro@gmail.com, giri

### Organizer 2: 
- Irvi Aini, Traveloka, Kubernetes Maintainer/Software Engineer, irvifa@gmail.com, irvifa

### Organizer 3: 
- Indrajati Widyatmoko, Kata.ai, Software Engineer, indrajati.widyatmoko@gmail.com, Indrajati

### Organizer 4: 
- Imre Nagi, Schoters, CTO, imre.nagi2812@gmail.com, imrenagi

### Sponsor 1
Biznet

### Sponsor 2
Tetrate

### Fiscal sponsor that will accept payments
Perkodi (https://perkodi.org/). Non profit organizations in Indonesia.

**Comment by imrenagi** [comment]
@irvifa I think we can mark `Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses` as done

**Comment by irvifa** [comment]
@imrenagi I don’t saw it in the docs that we have, can you please give the details in there? Thanks for the reminded dude 👍

**Comment by KTan1226** [comment]
Welcome @irvifa! Please update the issue as you complete the items on the checklist. Once everything is complete, we'll sign off on your team creating the event site.

**Comment by irvifa** [comment]
@KTan1226 By the way is it possible for us to use other media than Eventbrite?

**Comment by KTan1226** [comment]
Hi @irvifa. In 2020, we are requiring that all organizers use Eventbrite.

**Comment by irvifa** [comment]
@KTan1226 after more discussion we realize 1 thing: credit card in Indonesia is only used in 1% of the population, so if we're using Eventbrite it will be difficult.

**Comment by caniszczyk** [comment]
We should also reach out to @dio if he is interested in organizing/sponsoring

**Comment by KTan1226** [comment]
Hi @irvifa- No problem. We are flexible with not using Eventbrite in instances like this. Let me know if you have any further questions.

**Comment by isnuryusuf** [comment]
hi @irvifa @imrenagi lets talk for the event sponsorship

**Comment by dio** [comment]
Would love to help in any way possible, including sponsorship 🙂.

**Comment by irvifa** [comment]
@isnuryusuf and @dio probably you could email us for this one, thanks!

**Comment by irvifa** [comment]
This will be postponed until further notice 🙏 cc @girikuncoro

**Comment by girikuncoro** [comment]
this is due to growing Covid-19 situation across Indonesia as well, and we really want to do in-person conference, instead of online.
cc @imrenagi

**Comment by KTan1226** [comment]
Thanks for the update @irvifa- let me know if I can help with anything!

**Comment by xmulligan** [comment]
@irvifa We are relaunching the KCD program, including virtual events. We would love to have one in Jakarta if you are still interested! You can read [the blog post here](https://www.cncf.io/blog/2021/03/01/relaunching-kubernetes-community-days-with-kcd-africa-bengaluru/). If you have any questions, please contact me at kcd@cncf.io or [book a meeting with me](https://calendly.com/billmulligan/1-1-with-bill-mulligan?month=2021-03).
